src/lib.rs

```diff
@@ -150,6 +150,9 @@ pub fn middleware(app: Arc<App>) -> MiddlewareBuilder {
     m.add(conduit_cookie::Middleware::new(app.session_key.as_bytes()));
     m.add(conduit_cookie::SessionMiddleware::new("cargo_session",
                                                  env == Env::Production));
+    if env == Env::Production {
+        m.add(http::SecurityHeadersMiddleware);
+    }
```

Comment:
Since these headers were picked up when this was deployed on staging it looks like the middleware layers are stacked in the correct order.

Given that in production most static assets are served directly from nginx, I think `X-Content-Type-Options: nosniff` should also be included in `nginx.conf.erb`.   Looking over the other headers, I don't think any of the rest apply to such assets (except HSTS).